cypress/integration/ContextMenu.spec.ts

```diff
@@ -69,18 +69,21 @@ describe('ContextMenu', () => {
         // Disabled item
         pointerOver('Bookmarks →');
         cy.findByText('Inbox').should('be.visible');
+        pointerExitRightToLeft('Bookmarks →');
```

Comment:
Curious why we're adding these new `pointerExitRightToLeft` steps everywhere?

Reply:
I removed the `focus` call in pointer utils because it was technically not an accurate representation of the interaction and I wanted to make sure it hadn't masked any inaccurate tests.   It ended up failing these two because the exit grace works so well in the story that the sibling items are still within the zone (using default coords) so the menu never closes.

Anyway,   I added the directional exits to place this outside the zone before moving to the item, but your comment made me look at it again and I realised this test isn’t actually that helpful as users cant “hop” across items like this in the real world. I’ve instead consolidated them to cover a more helpful scenario while avoiding the need to handle this https://github.com/radix-ui/primitives/pull/802/commits/5d492779644b6f309f6b78aa728d6bb21e0611f7